changelogs/current.yaml

```diff
@@ -334,19 +374,38 @@ new_features:
     added :ref:`matcher <arch_overview_rbac_matcher>` for selecting connections and requests to different actions.
 - area: http
   change: |
-    added :ref:`treat_missing_header_as_empty <envoy_v3_api_field_config.route.v3.HeaderMatcher.treat_missing_header_as_empty>` to allow header match rule to treat the header value as empty and apply the match rule when the header is missing.
+    added :ref:`treat_missing_header_as_empty <envoy_v3_api_field_config.route.v3.HeaderMatcher.treat_missing_header_as_empty>`
+    to allow header match rule to treat the header value as empty and apply the match rule when the header is missing.
 - area: thrift
   change: |
-    added ``validate_clusters`` in :ref:`RouteConfiguration <envoy_v3_api_msg_extensions.filters.network.thrift_proxy.v3.RouteConfiguration>` to override the default behavior of cluster validation.
+    added ``validate_clusters`` in :ref:`RouteConfiguration <envoy_v3_api_msg_extensions.filters.network.thrift_proxy.v3.RouteConfiguration>`
+    to override the default behavior of cluster validation.
 - area: admin
   change: |
     added compile-time option ``--define=admin_html=disabled`` to disable HTML home page.
 - area: router
   change: |
-    added :ref:`ignore_port_in_host_matching <envoy_v3_api_field_config.route.v3.RouteConfiguration.ignore_port_in_host_matching>`. When set to true, port number (if any) in host header is ignored during host matching.
+    added :ref:`ignore_port_in_host_matching <envoy_v3_api_field_config.route.v3.RouteConfiguration.ignore_port_in_host_matching>`. When set to ``true``,
+    port number (if any) in host header is ignored during host matching.
 - area: router
   change: |
-    added :ref:`ignore_path_parameters_in_path_matching <envoy_v3_api_field_config.route.v3.RouteConfiguration.ignore_path_parameters_in_path_matching>`. When set to true, path-parameters(`rfc1808 <https://datatracker.ietf.org/doc/html/rfc1808>`_) is ignored during path matching.
+    added :ref:`ignore_path_parameters_in_path_matching <envoy_v3_api_field_config.route.v3.RouteConfiguration.ignore_path_parameters_in_path_matching>`.
+    When set to true, path-parameters(`rfc1808 <https://datatracker.ietf.org/doc/html/rfc1808>`_) is ignored during path matching.
+    added :ref:`ignore_path_parameters_in_path_matching <envoy_v3_api_field_config.route.v3.RouteConfiguration.ignore_path_parameters_in_path_matching>`.
+    When set to ``true``, path-parameters(rfc1808) is ignored during path matching.
+- area: docs
+  change: |
+    added warning for single backtick use in API docs and cleaned up most of the current problems. This will become an error soon.
```

Comment:
Do you think this should be part of the release notes?
IIUC it's more of an internal developers style issue, and not impacting users.

Reply:
well it impacts devs - that was my thinking